models/UserSwitch.php

```diff
@@ -61,7 +61,7 @@ public function attributeLabels()
     public function getUser()
     {
         if (empty($this->user)) {
-            $this->user = \Yii::$app->user;
+            $this->user = Yii::$app->getUser();
```

Comment:
`getUser()` method may not exist, it is not part of `yii\base\Application`. Better leave it as is or use `->get('user');`